Patches/Medical System Expansion - Revived/HediffDefs/Hediffs_AddedParts/Patch_Hediffs_AddedParts_VanillaBionic.xml

```diff
@@ -106,20 +106,34 @@
 				</value>
 			</li>
 			<!-- PowerClaw -->
-			<li Class="PatchOperationAdd">
-				<xpath>Defs/HediffDef[defName="PowerClaw"]</xpath>
-				<value>
-					<stages>
-						<li>
-							<minSeverity>0.000001</minSeverity>
-							<statOffsets>
-								<ArmorRating_Sharp>0.038</ArmorRating_Sharp>
-								<ArmorRating_Blunt>0.069</ArmorRating_Blunt>
-								<ArmorRating_Heat>0.007</ArmorRating_Heat>
-							</statOffsets>
-						</li>
-					</stages>
-				</value>
+			<li Class="PatchOperationConditional">
```

Comment:
I'm not too familiar with this mod. If I'm reading this correctly, the "powerclaw" gives armor value to the pawn using it? Is it just a limb, or some kind of full-body thing?

Reply:
The power claw is a limb replacement (same as vanilla), however I did not find a way to apply armor to a specific part of the body. I wanted the patched hediffs to have armor offsets - same as unpatched, however it would not have made sense for a bionic arm to offer the same amount of protection as a bionic hand, as such the armor bonuses were calculated based off bodypart coverage.